tests/org.jboss.tools.central.ui.bot.test/src/org/jboss/tools/central/test/ui/reddeer/LauncherApplicationTest.java

```diff
@@ -0,0 +1,208 @@
+ ******************************************************************************/
+package org.jboss.tools.central.test.ui.reddeer;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.fail;
+
+import java.util.List;
+import java.util.regex.Pattern;
+
+import org.eclipse.reddeer.common.logging.Logger;
+import org.eclipse.reddeer.common.wait.TimePeriod;
+import org.eclipse.reddeer.common.wait.WaitWhile;
+import org.eclipse.reddeer.eclipse.core.resources.Project;
+import org.eclipse.reddeer.eclipse.ui.navigator.resources.ProjectExplorer;
+import org.eclipse.reddeer.eclipse.ui.problems.Problem;
+import org.eclipse.reddeer.eclipse.ui.views.markers.ProblemsView;
+import org.eclipse.reddeer.eclipse.ui.views.markers.ProblemsView.ProblemType;
+import org.eclipse.reddeer.junit.runner.RedDeerSuite;
+import org.eclipse.reddeer.swt.impl.shell.DefaultShell;
+import org.eclipse.reddeer.swt.impl.toolbar.DefaultToolItem;
+import org.eclipse.reddeer.workbench.core.condition.JobIsRunning;
+import org.eclipse.reddeer.workbench.handler.WorkbenchShellHandler;
+import org.eclipse.reddeer.workbench.impl.editor.DefaultEditor;
+import org.eclipse.reddeer.workbench.impl.shell.WorkbenchShell;
+import org.jboss.tools.central.reddeer.api.ExamplesOperator;
+import org.jboss.tools.common.launcher.reddeer.wizards.NewLauncherProjectWizard;
+import org.jboss.tools.common.launcher.reddeer.wizards.NewLauncherProjectWizardPage;
+import org.json.simple.JSONArray;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ *
+ * @author zcervink
+ * @see https://issues.jboss.org/browse/JBIDE-26520
+ *
+ */
+@RunWith(RedDeerSuite.class)
+public class LauncherApplicationTest {
+
+	private static final String CENTRAL_LABEL = "Red Hat Central";
+	private static final Logger log = Logger.getLogger(ExamplesOperator.class);
+	private static final String BLACKLIST_ERRORS_REGEXES_FILE = "resources/blacklist-launcher-application-errors-regexes.json";
+
+	@BeforeClass
+	public static void setup() {
+		new DefaultToolItem(new WorkbenchShell(), CENTRAL_LABEL).click();
+		// activate central editor
+		new DefaultEditor(CENTRAL_LABEL);
+	}
+
+	@After
+	public void teardown() {
+		WorkbenchShellHandler.getInstance().closeAllNonWorbenchShells();
+		for (Project p : new ProjectExplorer().getProjects()) {
+			p.delete(true);
+		}
+		new DefaultToolItem(new WorkbenchShell(), CENTRAL_LABEL).click();
+		// activate central editor
+		new DefaultEditor(CENTRAL_LABEL);
+	}
+
+	@Test
+	public void createNewLauncherApplicationProjectWithDefaultSettings() {
+		log.step("Import project start");
+		
+		NewLauncherProjectWizard wizard = new NewLauncherProjectWizard();
+		NewLauncherProjectWizardPage wizardPage = new NewLauncherProjectWizardPage("New Launcher project");
+		
+		wizard.openWizardFromShellMenu();
+		wizardPage.setProjectName("project_1_default_settings");
+
+		wizardPage.clickTheFinishButton();
```

Comment:
As was mentioned in your reddeer test-framework plugin, finish method should be implemented over wizard dialog object, not wizardpage object that accommodates form page object like labels and set up button. wizard dialog is suitable for control button like finish, next, etc...